quickstarts/kubernetes/google-kubernetes-engine/alerts/HighDiskUsage.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: High CPU Utilization
+
+details: |+
+  This alert is triggered when the memory usage is above 85% of total capacity.
+
+type: STATIC
+nrql:
+  query: "SELECT max(cpuPercent) FROM SystemSample WHERE clusterName LIKE '%'"
```

Comment:
@EmilHammarstrand See [this comment](https://github.com/newrelic/newrelic-quickstarts/pull/942/files#r843088971)